src/node.h

```diff
@@ -375,8 +375,6 @@ NODE_DEPRECATED("Use WinapiErrnoException(isolate, ...)",
 })
 #endif
 
-const char *signo_string(int errorno);
```

Comment:
*If* I’m right about that, moving `signo_string` to a different namespace would be a breaking ABI change, too.

I’m not sure what the performance impact of not being able to inline these functions is, but moving the function bodies into their own `.cc` files may be more appropriate anyway (like they are now in `node.cc`). If you do that, you should probably drop any `static`/`inline` qualifiers (Note how `signo_string` doesn’t have one right now, either).